src/components/ShowTable/ShowTable.css

```diff
@@ -1,15 +1,20 @@
+
+
 .ShowTable{
   position: absolute;
   left: 0;
   width: 100%;
-  height: 50%;
+  height: 82%;
```

Comment:
Use css calc() to do this
height: calc(100% - var(--Header-height) - var(--NavBar-height));